<ui version="4.0" >
 <class>LifetimeProtocolDialog</class>
 <widget class="QDialog" name="LifetimeProtocolDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowIcon" >
   <iconset resource="application.qrc" >:/img/view_text.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QTableWidget" name="tableWidget" >
     <property name="columnCount" >
      <number>7</number>
     </property>
     <column>
      <property name="text" >
       <string>Date</string>
      </property>
      <property name="icon" >
       <iconset resource="application.qrc" >:/img/run.png</iconset>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Coming</string>
      </property>
      <property name="icon" >
       <iconset resource="application.qrc" >:/img/kommt.xpm</iconset>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Going</string>
      </property>
      <property name="icon" >
       <iconset resource="application.qrc" >:/img/geht.xpm</iconset>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Break (min)</string>
      </property>
      <property name="icon" >
       <iconset resource="application.qrc" >:/img/working_pause.png</iconset>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Target (h)</string>
      </property>
      <property name="icon" >
       <iconset resource="application.qrc" >:/img/configure.png</iconset>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Overtime</string>
      </property>
      <property name="icon" >
       <iconset resource="application.qrc" >:/img/sum.png</iconset>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Job Description</string>
      </property>
      <property name="icon" >
       <iconset resource="application.qrc" >:/img/view_text.png</iconset>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_insert" >
       <property name="toolTip" >
        <string>Add an empty row below the selected one</string>
       </property>
       <property name="text" >
        <string>Add row</string>
       </property>
       <property name="icon" >
        <iconset resource="application.qrc" >:/img/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete" >
       <property name="toolTip" >
        <string>Remove selected row(s)</string>
       </property>
       <property name="text" >
        <string>Remove row</string>
       </property>
       <property name="icon" >
        <iconset resource="application.qrc" >:/img/remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>171</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Overtime:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_overtime" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_reload" >
       <property name="toolTip" >
        <string>Recalculate the overtime</string>
       </property>
       <property name="icon" >
        <iconset resource="application.qrc" >:/img/reload.png</iconset>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_open" >
        <property name="toolTip" >
         <string>Open another protocol file in a new window</string>
        </property>
        <property name="text" >
         <string>Open</string>
        </property>
        <property name="icon" >
         <iconset resource="application.qrc" >:/img/fileopen.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_saveas" >
        <property name="toolTip" >
         <string>Save this protocol to a new file</string>
        </property>
        <property name="text" >
         <string>Save as</string>
        </property>
        <property name="icon" >
         <iconset resource="application.qrc" >:/img/filesaveas.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_import" >
        <property name="toolTip" >
         <string>Import data from another format</string>
        </property>
        <property name="text" >
         <string>Import Data...</string>
        </property>
        <property name="icon" >
         <iconset resource="application.qrc" >:/img/run.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_export" >
        <property name="toolTip" >
         <string>Export data to another format</string>
        </property>
        <property name="text" >
         <string>Export Data...</string>
        </property>
        <property name="icon" >
         <iconset resource="application.qrc" >:/img/run.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>321</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK" >
       <property name="toolTip" >
        <string>Save changes and close</string>
       </property>
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="icon" >
        <iconset resource="application.qrc" >:/img/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel" >
       <property name="toolTip" >
        <string>Do not save changes and close</string>
       </property>
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_reload</tabstop>
  <tabstop>pushButton_OK</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
  <tabstop>pushButton_insert</tabstop>
  <tabstop>pushButton_delete</tabstop>
  <tabstop>pushButton_open</tabstop>
  <tabstop>pushButton_saveas</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>lineEdit_overtime</tabstop>
 </tabstops>
 <resources>
  <include location="application.qrc" />
 </resources>
 <connections/>
</ui>
